### ISSUE TITLE
Add Mealplan feature

### ISSUE THREAD

**Issue by MrJohnZoidberg** [patzly/grocy-android#524]

**Comment by YasTouchedMySpaggett** [comment]
Hi, sorry for repeating the question. Will this be included in v3.0.0? Is it present on the beta build?

**Comment by dominiczedler** [comment]
Sorry for the late reply, no this feature is unfortunately built yet.

**Comment by hairy-tortoise** [comment]
Will this be released soon? @dominiczedler

**Comment by dominiczedler** [comment]
In the next weeks: no
In the next months: probably yes :)

I would like to have the feature in the app, too. So the chances are not bad.

**Comment by packman2008** [comment]
Adding meal planner functionality would be great, but for me at least just being able to view the meal plan created in the web interface would be a good start. A regular question for me is "What are we eating this evening?" and that could easily be answered by a quick look at the plan in the app.

**Comment by dominiczedler** [comment]
Short update :)

<img src="https://github.com/patzly/grocy-android/assets/27244574/df6d80ba-a068-4625-8c5c-ed85f012bc63" height="400" />

**Comment by BenBergman** [comment]
I can view a meal plan but I can't see how to add a meal to the plan. Is that feature still missing?

**Comment by patzly** [comment]
@BenBergman Yes, this feature is not implemented. You currently have to use the web interface for that.
